srcpkgs/dasel/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'dasel'
+pkgname=dasel
+version=1.24.1
+revision=1
+build_style=go
+go_import_path=github.com/tomwright/dasel
+go_package="${go_import_path}/cmd/dasel"
+go_ldflags="-X ${go_import_path}/internal.Version=${version}"
+short_desc="Query and modify data structures using selector strings"
+maintainer="Daniel Eyßer <daniel.eysser@gmail.com>"
+license="MIT"
+homepage="https://daseldocs.tomwright.me/"
+changelog="https://raw.githubusercontent.com/TomWright/dasel/v${version}/CHANGELOG.md"
```

Comment:
this should point to the changelog file on the main/master branch, not the tag

Reply:
done, thanks for the hint